models/Program.ts

```diff
@@ -0,0 +1,31 @@
+// TODO: weekday field got moved to class so we can move this to Class.ts after
+// export enum weekdays {
+//     MON = "MON",
+//     TUE = "TUE",
+//     WED = "WED",
+//     THU = "THU",
+//     FRI = "FRI",
+//     SAT = "SAT",
+//     SUN = "SUN",
+// }
```

Comment:
can we take this out now?

Reply:
We're going to move it to `Class` so there isn't any harm in keeping it here for now.